docs/self-signed-certificate.md

```diff
@@ -0,0 +1,24 @@
+# Self-signed certificate
+
+Velero starts to support installing default BackupStorageLocation with self-signed certificate from v1.4.2. More details please refer to https://velero.io/docs/v1.7/self-signed-certificates/.
+
+To keep consistency with velero, Velero plugin for vsphere supports using velero-plugin-for-vsphere with a storage provider secured by a self-signed certificate from v1.1.1.
```

Comment:
Would you please add a list of supportability in terms of cluster flavor here? It would be more straight-forward for users. For example,
- Vanilla cluster: supported since v1.1.1
- Guest cluster: supported since v1.1.1
-  Supervisor cluster: not supported

Reply:
+1